skillful_network_server/src/main/java/fr/uca/cdr/skillful_network/entities/user/Role.java

```diff
@@ -71,4 +86,22 @@ public boolean equals(Object o) {
 	public int hashCode() {
 		return Objects.hash(id, name);
 	}
+
+	public static HashMap<Name, String> getNamesAndDescriptions() {
+		HashMap<Name, String> names = new HashMap<>();
+		for (Name n : Name.values()) {
+			names.put(n, n.getDescription());
+		}
+		System.out.println(names);
+		return names;
+	}
+
+	public static HashSet<String> getRoleDescriptions() {
+		HashSet<String> descriptions = new HashSet<>();
+		for (Name n : Name.values()) {
+			descriptions.add(n.getDescription());
+		}
+		System.out.println(descriptions);
```

Comment:
Could you use a logger like in the `AuthenticationServiceImpl` : 
```java
private static final Logger logger = LoggerFactory.getLogger(AuthenticationService.class);
```

Please, could you do that for all your `System.out.println`?

It will outputs well-formatted lines. (and thus when in production, it will be added (not configured, in log files that can be stored and used later)